common/app/controllers/EmailSignupController.scala

```diff
@@ -36,7 +36,7 @@ object ListIds {
   val guardianTodayUs = 1493
   val guardianTodayAu = 1506
 
-  val theBestOfOpinion = 2313
+  val theBestOfOpinion = 3811
   val newBestOfOpinion = 3811
```

Comment:
Removing the check for `2313` will remove the Opinion tones from any embed using: https://www.theguardian.com/email/form/article/2313 
This may affect some composer content (they will go back to being blue).

- Do we want to keep supporting the old embed?

Mostly what this list does is set the tone used on the email form. Not all emails need to be added to it, unless you want a tone applied, or a triggered send (confirmation email).

- Line 40 is also responsible for adding tones to the new list ID (3811) Support for the 3811 iframe embed was added in #15488
- If you change the name, you will also need to change it in `emailSignUp.scala.html` (which is applying tones based on `newBestOfOpinion`)